launchpad.py

```diff
@@ -108,6 +108,28 @@ class LaunchpadPro(LaunchPad):
         mido.Message("sysex", data=[0, 32, 41, 2, 16, 10, 99, 0]),
     ]
 
+class LaunchpadX(LaunchPad):
+
+    palette = "RGB"
+    message2led = {}
+    led2message = {}
+    for row in range(1, 9):
+        for column in range(1, 9):
+            note = 10*row + column
+            message2led["NOTE", note] = row, column
+            led2message[row, column] = "NOTE", note
+    for i, button in enumerate(ARROWS + MENU):
+        control = 91 + i
+        message2led["CC", control] = button
+        led2message[button] = "CC", control
+
+    setup = [
+        # This SysEx message switches the LaunchPad Pro to "programmer" mode
+        mido.Message("sysex", data=[0, 32, 41, 2, 12, 14, 1]),
+        # And this one sets the front/side LED
```

Comment:
Does the LP X also have a "front" LED? (If not, I'll remove that sysex since it might be confusing :sweat_smile: )

Reply:
I wondered about that.
I cannot see one.
I have no other Launchpad to compare it to